internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -129,109 +129,6 @@ func (e *EnvStackConfig) Template() (string, error) {
 	return content.String(), nil
 }
 
-func (e *EnvStackConfig) vpcConfig() template.VPCConfig {
```

Comment:
These deletions of `EnvStackConfig` (`vpcConfig`, `importVPC`, ..., `internalALBSubnets`) are not real: I just moved further down in this file. Nothing is changed.